lib/puppetserver/ca/action/create.rb

```diff
@@ -14,21 +14,23 @@ class Create
 
         # Only allow printing ascii characters, excluding /
         VALID_CERTNAME = /\A[ -.0-~]+\Z/
-        CERTNAME_BLACKLIST = %w{--all --config}
+        CERTNAME_BLACKLIST = %w{--all --puppet-config --server-config}
 
         SUMMARY = "Create a new certificate signed by the CA"
         BANNER = <<-BANNER
 Usage:
   puppetserver ca create [--help]
-  puppetserver ca create [--config PATH] [--certname CERTNAME[,ADDLCERTNAME]]
+  puppetserver ca create [--puppet-config PATH] [--server-config PATH]
+                         [--certname CERTNAME[,ADDLCERTNAME]]
```

Comment:
I know this is a WIP.  But we'll need to update the run function below to support these changes.

Reply:
Oh yeah, guess I missed this one.